src/intl/messages.js

```diff
@@ -120,6 +120,8 @@ export const messages: { [messageId: string]: MessageType } = {
   },
   coresPerSockets: 'Cores per Virtual Socket',
   cpus: 'Total Virtual CPUs',
+  cpusBadTopology: 'No valid CPU topology exists for this total Virtual CPUs count.',
+  cpusBadTopologySelection: 'Selected vCPU topology is invalid.',
```

Comment:
It was somehow hidden between all posts but as mentioned on https://github.com/oVirt/ovirt-web-ui/pull/1489#issuecomment-891911743

> 2. Error handling for detected invalid CPU architecture:
> If we decide to leave the current solution of displaying an error then please rephrase the error to:
> "Selected **CPU** topology is invalid" (instead of vCPU)